tests/test_data/dummy_bootloaders/bootloader1

```diff
@@ -0,0 +1 @@
+i am fake bootloader data
```

Comment:
Nothing you did is wrong, however this woud be the python way of doing it: https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
I could be wrong, but it seems that fixtures documentation doesn’t talk about creating temp files. Can you clarify your suggestion here?

Reply:
@SchoolGuy please take a look at my update. It's working for me:
![image](https://user-images.githubusercontent.com/43251403/136138567-f092df93-5206-412b-8a69-9799dd4ebcc5.png)